templates/mautic/0/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2'
+services:
+  mautic:
+    image: mautic/mautic
```

Comment:
Don't use generic docker image tag, implicit neither explicit, they are not reproducible. Please use concrete docker image version. 
```
-    image: mautic/mautic
+    image: mautic/mautic:2.12-fpm
OR
+    image: mautic/mautic:2.12-apache
```